CMakeLists.txt

```
@@ -48,7 +48,7 @@ endif()
 include(Ccache)
 
 #
-if ("${CMAKE_SYSTEM_PROCESSOR}" MATCHES "(x86|AMD64)")
+if ("${CMAKE_SYSTEM_PROCESSOR}" MATCHES "(i386|i486|i586|i686|x86|AMD64)")
```

If my memories are right SSE appeared with Pentium/i586

```suggestion
if ("${CMAKE_SYSTEM_PROCESSOR}" MATCHES "(i586|i686|x86|AMD64)")
```